src/register-org/components/document-exchange-reference/document-exchange-reference.component.ts

```diff
@@ -46,13 +46,18 @@ export class DocumentExchangeReferenceComponent extends RegisterComponent implem
         // Set corresponding registration data
         this.registrationData.hasDxReference = false;
         // Navigate to office address page
-        this.router.navigate([this.registerOrgService.REGISTER_ORG_NEW_ROUTE, 'office-addresses']);
+        this.router.navigate([this.registerOrgService.REGISTER_ORG_NEW_ROUTE, 'regulatory-organisation-type']);
       }
     }
   }
 
   public onBack(): void {
-    this.navigateToPreviousPage();
+    const previousUrl = this.currentNavigation?.previousNavigation?.finalUrl?.toString();
+    if (previousUrl.includes(this.registerOrgService.CHECK_YOUR_ANSWERS_ROUTE)) {
+      this.router.navigate([this.registerOrgService.REGISTER_ORG_NEW_ROUTE, this.registerOrgService.CHECK_YOUR_ANSWERS_ROUTE]);
+    } else {
+      this.router.navigate([this.registerOrgService.REGISTER_ORG_NEW_ROUTE, 'registered-address', 'external']);
```

Comment:
should be 'internal'

Reply:
This will be handled as part of https://tools.hmcts.net/jira/browse/EUI-8835 as the testing of this PR is already complete.